conf/modules.config

```diff
@@ -637,27 +581,32 @@ process{
 process{
 
     withName: 'MERGEMUTECTSTATS' {
-        ext.prefix       = { "${meta.sample}.vcf.gz" }
+        ext.prefix       = { "${meta.id}.vcf.gz" }
     }
     withName: 'GATHERPILEUPSUMMARIES' {
-        ext.prefix       = { "${meta.sample}.table" }
+        ext.prefix       = { "${meta.id}.table" }
```

Comment:
For one ID we could have multiple samples, like tumor & recurrent,  would they be overwritten?

Reply:
Good point, but we haven't fully refactored mutect2 yet, I was just doing some quick replace

Reply:
But shouldn't the id be unique in any case?
For multiple sample we have the meta.patient, right?